editor/js/Viewport.js

```diff
@@ -413,6 +413,7 @@ var Viewport = function ( editor ) {
 
 	signals.objectRemoved.add( function ( object ) {
 
+		controls.enabled = true;
```

Comment:
Maybe adding the issue would clarify why this line is now present. Something like
```
controls.enabled = true; // #14810
```